projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -215,12 +215,16 @@ const ProductCard = props => {
 	/**
 	 * Calls the passed function onActivate after firing Tracks event
 	 */
-	const activateHandler = useCallback( () => {
-		recordEvent( 'jetpack_myjetpack_product_card_activate_click', {
-			product: slug,
-		} );
-		onActivate();
-	}, [ slug, onActivate, recordEvent ] );
+	const activateHandler = useCallback(
+		event => {
+			event.preventDefault();
```

Comment:
Awesome, thanks for adding these `event.preventDefault()`'s.  It fixes a prior existing bug where the page was unnecessarily reloading after clicking the "Activate" button. 🙌